about/pricing.html.markerb

```diff
@@ -39,11 +39,13 @@ Resources included for free on all plans:
 
 Additional resources are billed at the usage-based pricing detailed below.
 
-## _Compute_
+## Compute
 
 ### Fly Machines
 
-Pricing for [Fly Machines](/docs/machines/) VMs, which are also the virtual-machine building blocks of [Fly Launch](/docs/apps/) and [Fly Postgres](/docs/postgres/) apps.
+The price of a [Fly Machine](/docs/machines/) VM is the price of a preset CPU/RAM combination, plus about $5 per 30 days per GB of additional RAM.
+
+The available CPU/RAM combinations and their pricing are as follows:
 
 <%= partial("shared/cpu_mem_machines_pricing") %>
```

Comment:
Will this still show the current matrix? Because it includes combos that aren't "presets" as defined above. The only presets are:

shared-1x-256mb, shared-2x-512mb, shared-4x-1gb, shared-8x-2gb
perf-1x-2gb, perf-2x-4gb, perf-4x-8gb, perf-8x-16gb, perf-16x-32gb

Reply:
I can see reading [the matrix](https://fly.io//docs/about/pricing/#compute) as either "each row is a preset" or "the first of each grouping is the preset."

I think you reach the same bottom-line either way, but maybe if the preset => base RAM is very obvious (or the only thing on the matrix) the invoices will connect a little more.

I'm not sure!

Reply:
How about now? I put "named" in front of "presets" and made it explicit that the additional RAM configs are in the table.